app/assets/stylesheets/components/_discussion.css.sass

```diff
@@ -1,13 +1,16 @@
-$discussion-item-triangle-size: 40px
+$discussion-marker-border-width: 3px
```

Comment:
Another thing we try to enforce is the variable naming. We try to convey meaning by setting the most generic parts in the left and getting to the most specific to the right.
So this particular case could become:
`$discussion-border-width-marker`.
Also since using dashes alone and trying to create groupings in the name do not go well together, using camelCase as well is a good idea, so:
`$discussion-borderWidth-marker`.